### ISSUE TITLE
Does it works on win10?

### ISSUE THREAD

**Issue by ihqsat** [h4ck-rOOt/Lynda-Decryptor#37]
I am trying to use it on win10 but the destination folder keeps empty with, apparently, successful decryption.
Thank you.
![image](https://user-images.githubusercontent.com/32584070/31302033-7546c666-aaf6-11e7-910f-91d7bae82401.png)

**Comment by Dev-iL** [comment]
Yes, it works very well on Win 10. You might want to show the actual command that you're running - perhaps this is an indication of a completely different problem.

**Comment by ihqsat** [comment]
On windowns 10 cmd, after 'cd' to the path where it is the LyndaDecryptor.exe, I run 'LyndaDecryptor /D "C:\Users\MyUserName\AppData\Local\Packages\lynda.com.lynda.com_0dmhem0sv94sr\LocalState\offline\100222"'. Then, it creates a folder, "decrypted", in the ...\offline\ directory but empty.

When I try to run with the /F option:
'LyndaDecryptor /F "C:\Users\MyUserName\AppData\Local\Packages\lynda.com.lynda.com_0dmhem0sv94sr\LocalState\offline\100222\111245_0b79795d3efc95b9976c7c5b933afce2_67059051.ldcw" "C:\Users\MyUserName\Desktop\videos"                                                                                                           [ARGS] Changing mode to Single decryption!                                                                              [START] Init Decryptor...                                                                                               [START] Decryptor successful initalized!                                                                                                                                                                                                        [ERR] Couldn't load file: C:\Users\MyUserName\AppData\Local\Packages\lynda.com.lynda.com_0dmhem0sv94sr\LocalState\offline\100222\111245_0b79795d3efc95b9976c7c5b933afce2_67059051.ldcw                                                                                                                                                                                    Press any key to exit the program... '
I get this error...

I also tryed without parentheses and with other folders and nothing worked.
Thank you for responding.

**Comment by Dev-iL** [comment]
Ah, the issue is that you're trying to decrypt `. ldcw` files. These are
not supported by the decryptor _at this time_, **however**, somebody
suggested a solution here:
https://github.com/h4ck-rOOt/Lynda-Decryptor/issues/12#issuecomment-330521448
.

**Comment by Rohitc9537** [comment]
I have same problem with my windows 10
but solved it
just check whether your Decryption is completed or not. If completed just check log of output in command line
and check where the Decrypted files are located. 
![untitled](https://user-images.githubusercontent.com/21275597/31581500-43373d7c-b18b-11e7-8770-907714f2f558.png)

basically it stores in same folder where lynda encrypted files are stored

**Comment by DmitriySokhach** [comment]
As LDWC merge request will be merged, I suggest to dedicate this issue to improving **Decryption completed!** message to something like:
On finishing: "**Done. XX of XX files were decrypted.**"

**Comment by InkWithGold** [comment]
Can someone help, after starting cmd as administrator and pointing to the unzipped folder then running

[[

LyndaDecryptor /D "C:\Lynda\ldc_dl_courses\[course#]" /DB "C:\Users\[my user]\AppData\Local\lynda.com\Lynda.com Desktop App\db.sqlite" /OUT "C:\Lynda\ldc_dl_courses\[course#]\D"

]]

I get
('LyndaDecryptor' is not recognized as an internal or external command, operable program or batch file.)

**Comment by DmitriySokhach** [comment]
"Lynda-Decryptor-master" seems to me that you have downloaded the source code, but should have download release archive from: [https://github.com/h4ck-rOOt/Lynda-Decryptor/releases](url)

**Comment by InkWithGold** [comment]
yep.. seems i was linked to the source code instead of the release. To my surprize, working now. Thank you

**Comment by Ravin1822** [comment]
I am not able to decrypt .LDCW files. please help me out.
C:\Users\Ravin\Downloads\LyndaDecryptor>LyndaDecryptor /D "C:\Users\Ravin\AppData\Local\Packages\lynda.com.lynda.com_0dmhem0sv94sr\LocalState\offline\419164" /DB "C:\Users\Ravin\AppData\Local\Packages\lynda.com.lynda.com_0dmhem0sv94sr\LocalState\db.sqlite"  /OUT "F:\offline\1"
[ARGS] Changing mode to Folder decryption!
[DB] Creating db connection...
[DB] DB successfully connected and opened!

[START] Init Decryptor...
[START] Decryptor successful initalized!

Decryption completed!

Press any key to exit the program...

**Comment by DmytroSokhach** [comment]
@Ravin1822 Are you trying with my temporary release 1.3.1 or with existing 1.3?
Try downloading from here: [https://github.com/DmytroSokhach/Lynda-Decryptor/releases/tag/1.3.1](url)
